DigitalLearningSolutions.Data/ViewModels/UserCentreAccount/UserCentreAccountsRoleViewModel.cs

```diff
@@ -0,0 +1,60 @@
+﻿using DigitalLearningSolutions.Data.Enums;
```

Comment:
If this is a ViewModel, it should be in the Web project at Web/ViewModels/ with all of the others, no?

Reply:
@kevwhitt-hee I followed "chooseACentre" view as a reference to build "UserCentreAccountsRole" page. I could see similar model class is available under "Data\ViewModels\" folder in Data.csproj file. Please confirm if we are good to move "UserCentreAccountsRoleViewModel" to "Web/ViewModels/".

Reply:
That makes sense, let's leave it where it is, then.